src/parser-python.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+const spawnSync = require("child_process").spawnSync;
+const path = require("path");
+
+function parseText(text, pythonExecutable) {
+  const executionResult = spawnSync(
+    pythonExecutable,
+    [path.join(__dirname, "../vendor/python/astexport.py")],
+    {
+      input: text
+    }
+  );
+
+  const error = executionResult.stderr.toString();
+
+  if (error) {
+    throw new Error(error);
+  }
+
+  return executionResult;
+}
+
+function parse(text, parsers, opts) {
+  const pythonExectuable = `python${opts.pythonVersion}`;
```

Comment:
Do we want do something like:

* try parse with `"python3"`,
* if error, then try parse with `"python"`

We actually do this with TypeScript ts and tsx.


Then we wouldn't need an option here.

Reply:
If not python version is specified it will use `python` directly, which means that will use the default python in your current environment, could be python2 or python3 depending on your setup (eg if you are in a virtualenv).

The option is mainly to run tests with both python2 and python3 :)

Reply:
Ok cool. So we don't need to document it.